tests/config/run_test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -xe
+
+# Check for a command-line argument
+if [ $# -eq 0 ]; then
+  echo "Error: Please provide a base path as a command-line argument."
+  exit 1
+fi
+
+function clean() {
+    rm -f "${base_path}/conf_test.conf"
+    rm -f "${base_path}/clvk.conf"
+}
+trap clean EXIT
+
+# Assign the base path from the argument
+base_path=$1
+
+# Create the first file with the base path
+echo "cache_dir=testing" > "${base_path}/conf_test.conf"
+
+# Create the second file with the base path
+echo "cache_dir=failed" > "${base_path}/clvk.conf"
+echo "compiler_temp_dir=not/overwritten/" > "${base_path}/clvk.conf"
+
+
+export CLVK_CONFIG_FILE="${base_path}/conf_test.conf"
+
+# Run test
+"${base_path}/config_test"
```

Comment:
```suggestion
# Run test
CLVK_CONFIG_FILE="${base_path}/conf_test.conf" \
"${base_path}/config_test"
```
Because one may want to run it locally, let's avoid exporting.